test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -1035,13 +1037,32 @@
             :ssl-crl-path (str bootstrap/server-conf-dir "/ssl/crl.pem")}
            :certificate-authority
            {:allow-auto-renewal true}}
-          (let [response (http-client/post
+          (let [generated-cert-info (generate-and-sign-a-cert! "foobar")
+                signed-cert-file (ks/temp-file)
+                _ (spit signed-cert-file (:signed-cert generated-cert-info))
+                _ (Thread/sleep 1000) ;; ensure some time has passed so the timestamps are different
```

Comment:
Can we generate a signed cert that is already old instead of sleeping in our tests? (Genuine Q, not sure if we need to exercise the http api or if calling the library functions allows us the flexibility to make the signed cert old)

Reply:
Maybe. I think we want a suite of acceptance (beaker level) tests around this new functionality, perhaps that could wait for those tests?